src/AutoRest.CSharp/Common/Output/Expressions/Snippets.Optional.cs

```diff
@@ -42,7 +41,13 @@ public static BoolExpression IsDefined(TypedValueExpression value)
             }
 
             public static ValueExpression ToDictionary(ValueExpression dictionary) => new InvokeStaticMethodExpression(Configuration.ApiTypes.OptionalType, Configuration.ApiTypes.OptionalToDictionaryName, new[] { dictionary });
-            public static ValueExpression ToList(ValueExpression collection) => new InvokeStaticMethodExpression(Configuration.ApiTypes.OptionalType, Configuration.ApiTypes.OptionalToListName, new[] { collection });
+
+            public static ValueExpression ToList(TypedValueExpression collection)
```

Comment:
Since it's not calling `Optional.ToList` anymore, shall we give this a better name? such as `FallbackToChangeTrackingList`